packages/flutter_tools/test/integration.shard/test_data/gen_l10n_project.dart

```diff
@@ -286,6 +288,16 @@ void main() {
       "hello": {},
       "world": {}
     }
+  },
+
+  "singleQuote": "Flutter's amazing!",
+  "@singleQuote": {
+    "description": "Love Flutter!"
```

Comment:
In most of the rest of this file, the message descriptions briefly describe what we're checking.

Reply:
Modified it to describe the test